json-endpoints/jetpack/class.jetpack-json-api-themes-install-endpoint.php

```diff
@@ -14,6 +14,12 @@ protected function install() {
 
 		foreach ( $this->themes as $theme ) {
 
+			// hook to allow alternative install method
+			if ( wp_endswith( $theme, '-wpcom' )
```

Comment:
If we wanted this to be more generic, we could remove the first condition `wp_endswith( $theme, '-wpcom' )`.

What do you think?

Otherwise, I would remove the above comment. It doesn't make sense (IMO) to say "allow alternative install method) when we are explicitly filtering for themes ending in `-wpcom`.